src/test/java/io/debezium/connector/yugabytedb/common/TestBaseClass.java

```diff
@@ -312,6 +381,50 @@ protected void waitAndFailIfCannotConsume(List<SourceRecord> records, long recor
     waitAndFailIfCannotConsume(records, recordsCount, 300 * 1000 /* 5 minutes */);
   }
 
+  protected void insertBulkRecords(int numRecords, String fullTableName) {
+    String formatInsertString = "INSERT INTO " + fullTableName + " VALUES (%d);";
+    TestHelper.executeBulk(formatInsertString, numRecords, DEFAULT_COLOCATED_DB_NAME);
+  }
+
+  /**
+   * Helper function to create the required tables in the database DEFAULT_COLOCATED_DB_NAME
+   */
+  protected void createTables(boolean colocation) {
```

Comment:
I think moving these methods out of the snapshot class won't make much sense since these tables are very specific to the snapshot tests only. Moving it to the `TestBaseClass` would mean that they can be used generically.

Maybe we can do one thing, rename the method to `createTablesInColocatedDB` and the next method to `dropTablesInColocatedDB` - that will ensure the method's name conveys what it is doing.

Similar renaming should be done for the insert methods as we have a method with a conflicting name in the class `TestHelper`.